src/api.js

```diff
@@ -1,16 +1,29 @@
+import { logError } from '@edx/frontend-platform/logging';
 import { examRequiresAccessToken, store } from './data';
 
 export function isExam() {
   const { exam } = store.getState().examState;
+  if (!exam) {
+    return false;
+  }
   return exam.id !== null;
 }
 
 export function getExamAccess() {
-  const { examAccessToken } = store.getState().examState;
+  const { exam, examAccessToken } = store.getState().examState;
+  if (!exam) {
+    logError('Failed to get exam access token. Not an exam.');
+    return '';
```

Comment:
you can just use `return;` here!

Reply:
When I tried to return, I got an error (I believe code quality though I can replicate to see what the error is) that these would result in inconsistent types, since the non-error case returns a string.